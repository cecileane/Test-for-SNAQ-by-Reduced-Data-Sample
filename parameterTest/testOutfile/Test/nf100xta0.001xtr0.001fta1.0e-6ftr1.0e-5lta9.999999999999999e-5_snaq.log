optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-6,
                       xtolAbs=0.001, xtolRel=0.001.
 max number of failed proposals = 100, liktolAbs = 9.999999999999999e-5.
rootname for files: nf100xta0.001xtr0.001fta1.0e-6ftr1.0e-5lta9.999999999999999e-5_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 81319
seed: 81319 for run 1
Sat Oct  1 17:22:04 2016

 BEGIN SNaQ for run 1, seed 81319 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 124 (failures: 100) with -loglik=126.74251 and ht_min=[0.2856,1.51213,0.48942,0.51916,10.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 28		 28		 1	 |	 0		 0		 --
mvtarget	 26		 26		 0	 |	 0		 0		 --
chdir		 6		 6		 2	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 63		 48		 1	 |	 4		 4		 --
Total		 124		 109		 5	 |	 4		 4		 4
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 126.74250981978506
((((5,(6,#H7:::0.28560302259462383):10.0):0.5191611909484348,3):0.48941855046849186,4):1.5121327149799382,2,(1)#H7:::0.7143969774053762);
---------------------
seed: 93370 for run 2
Sat Oct  1 17:22:13 2016

 BEGIN SNaQ for run 2, seed 93370 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 153 (failures: 100) with -loglik=126.74251 and ht_min=[0.2856,1.51214,0.51916,0.48942,10.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 37		 37		 1	 |	 2		 2		 --
mvtarget	 34		 34		 1	 |	 2		 2		 --
chdir		 15		 15		 2	 |	 4		 4		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 66		 50		 0	 |	 1		 1		 --
Total		 153		 137		 5	 |	 9		 9		 5
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 0.6

 FINISHED SNaQ for run 2, -loglik of best 126.74250982241327
(5,((4,(2,(1)#H7:::0.7143973784783482):1.5121421780124151):0.4894202301795226,3):0.5191602821534134,(6,#H7:::0.28560262152165183):9.999999411581108);
---------------------

Sat Oct  1 17:22:23 2016
MaxNet is ((((5,(6,#H7:::0.28560302259462383):10.0):0.5191611909484348,3):0.48941855046849186,4):1.5121327149799382,2,(1)#H7:::0.7143969774053762); 
with -loglik 126.74250981978506
