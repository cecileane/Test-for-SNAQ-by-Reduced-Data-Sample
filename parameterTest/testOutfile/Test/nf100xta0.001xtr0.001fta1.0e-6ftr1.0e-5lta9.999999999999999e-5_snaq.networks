((((5,(6,#H7:::0.28560302259462383):10.0):0.5191611909484348,3):0.48941855046849186,4):1.5121327149799382,2,(1)#H7:::0.7143969774053762);, with -loglik 126.74250981978506 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
((((5,(6)#H7:::0.7055255628276642):1.462296946618228,3):0.4626369636399155,4):0.6690960299851368,2,(1,#H7:::0.29447443717233573):9.987870434943376);, with -loglik 128.4994777272266
(#H7:::0.3988178706448965,2,(1,(6,(((4)#H7:::0.6011821293551035,3):0.03441564095455802,5):0.1994537957562078):1.169031188755527):2.297908846596373);, with -loglik 172.90601027204642
((#H7:::0.4271272351936619,4):1.7977797424335817,(5,(6,(1,(2)#H7:::0.5728727648063381):5.221069417957995):0.15863830981321153):0.03457769109553748,3);, with -loglik 191.9021185308895
(((3)#H7:::0.5973891432438483,4):0.07447578342798447,2,(1,(6,(#H7:::0.4026108567561517,5):9.954595604223789):0.991031291826463):0.0952649946513958);, with -loglik 215.12725804823242
(((#H7:::0.48256290354270326,3):0.9345788541507739,4):0.21648032672745915,2,(1,(6,(5)#H7:::0.5174370964572967):3.4319935135439197):0.015240017132743634);, with -loglik 234.81019178741434
