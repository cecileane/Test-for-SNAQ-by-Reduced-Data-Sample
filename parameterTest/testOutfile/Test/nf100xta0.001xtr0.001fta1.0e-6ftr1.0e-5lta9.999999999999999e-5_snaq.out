((((5,(6,#H7:::0.28560302259462383):10.0):0.5191611909484348,3):0.48941855046849186,4):1.5121327149799382,2,(1)#H7:::0.7143969774053762); -Ploglik = 126.74250981978506
 Dendroscope: ((((5,(6,#H7):10.0):0.5191611909484348,3):0.48941855046849186,4):1.5121327149799382,2,(1)#H7);
 Elapsed time: 18.708271325 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((((5,(6,#H7:::0.28560302259462383):10.0):0.5191611909484348,3):0.48941855046849186,4):1.5121327149799382,2,(1)#H7:::0.7143969774053762);, with -loglik 126.74250981978506
 (5,((4,(2,(1)#H7:::0.7143973784783482):1.5121421780124151):0.4894202301795226,3):0.5191602821534134,(6,#H7:::0.28560262152165183):9.999999411581108);, with -loglik 126.74250982241327
-------
